client/rest-high-level/src/test/java/org/elasticsearch/client/indices/AnalyzeResponseTests.java

```diff
@@ -19,25 +19,24 @@
 
 package org.elasticsearch.client.indices;
 
+import com.carrotsearch.randomizedtesting.annotations.Repeat;
+
 import org.elasticsearch.action.admin.indices.analyze.AnalyzeAction;
 import org.elasticsearch.client.AbstractResponseTestCase;
 import org.elasticsearch.common.xcontent.XContentParser;
 
 import java.io.IOException;
-import java.util.ArrayList;
 import java.util.Arrays;
-import java.util.HashMap;
-import java.util.List;
-import java.util.Map;
 
+@Repeat(iterations = 100)
 public class AnalyzeResponseTests extends AbstractResponseTestCase<AnalyzeAction.Response, AnalyzeResponse> {
 
     @Override
     protected AnalyzeAction.Response createServerTestInstance() {
         int tokenCount = randomIntBetween(1, 30);
         AnalyzeAction.AnalyzeToken[] tokens = new AnalyzeAction.AnalyzeToken[tokenCount];
         for (int i = 0; i < tokenCount; i++) {
-            tokens[i] = randomToken();
+            tokens[i] = org.elasticsearch.action.admin.indices.analyze.AnalyzeResponseTests.randomToken();
```

Comment:
Can you use a static import here?

Reply:
Actually I had to move this the random token generation to RandomObjects in `:test:framework` so it is usable from both server and client side response testing.